filament/include/filament/IndirectLight.h

```diff
@@ -342,6 +343,68 @@ class UTILS_PUBLIC IndirectLight : public FilamentAPI {
     /** @deprecated use static versions instead */
     UTILS_DEPRECATED
     math::float4 getColorEstimate(math::float3 direction) const noexcept;
+
+    /**
+     * Sets IBL lookup options.
+     *
+     * @param options Options for IBL lookup settings.
+     */
+    void setIblOptions(IblOptions const& options) noexcept;
+
+    /**
+     * Gets IBL lookup options.
+     *
+     * @return IBL lookup options.
+     */
+    IblOptions const& getIblOptions() const noexcept;
+
+    /**
+     * Sets IBL type.
+     *
+     * @param iblTechnique Determine the IBL type (infinite, or finite box/sphere)
+     */
+    void setIblTechnique(const IblOptions::IblTechnique iblTechnique) noexcept;
+
+    /**
+     * Gets IBL type.
+     */
+    IblOptions::IblTechnique getIblTechnique() const noexcept;
+
+    /**
+     * Sets the center of the finite IBL proxy geometry.
+     *
+     * @param iblCenter The center of the finite IBL proxy geometry
+     */
+    void setIblCenter(const math::float3& iblCenter) noexcept;
+
+    /**
+     * Gets IBL center.
+     */
+    const math::float3& getIblCenter() const noexcept;
+
+    /**
+     * Sets the half extents of the finite IBL proxy geometry.
+     *
+     * @param iblCenter The half extents of the finite IBL proxy geometry
+     */
+    void setIblHalfExtents(const math::float3& iblHalfExtents) noexcept;
+
+    /**
+     * Gets IBL half extents.
+     */
+    const math::float3& getIblHalfExtents() const noexcept;
```

Comment:
Instead of `IblTechnique` I would simply use `IndirectLight::Mode`: `INFINITY`, `BOX`, `SPHERE`. 

I don't think we need a setter for the `Mode`, you could just have:

`void setSphereProxy(math::float4 sphere)`
`void setBoxProxy(Box const& box)`
`void clearProxy()`

`Mode getMode() const`
`math::float4 getSphereProxy()`
`Box const& getBoxProxy()`

Reply:
Fixed in b0f2ef543036f99a374befdcb44e38fbfa2b8df5 and 901f2a025aa0c1be79f2c56d51f7833d533bb4e4